.github/workflows/integration.yaml

```diff
@@ -370,3 +370,154 @@ jobs:
       - name: Run tests
         run: |
           cd $(go env GOROOT)/src/os; wazero run -mount=/:/ ~/bin/os.wasm -test.v
+
+  build_go_tests:
+    name: Build Go test binaries
+    runs-on: ubuntu-22.04
+    env:
+      BINARY_OUT: ./go-test-binaries
+
+    steps:
```

Comment:
you might want to checkout wazero _before_ the cache action in order to place the relative cache dir under wazero directory in the following jobs

Reply:
also @Pryz